src/PooledArrays.jl

```diff
@@ -377,6 +462,12 @@ function unsafe_pool_push!(pa::PooledArray{T,R}, val) where {T,R}
            )))
     end
     pool_idx = convert(R, _pool_idx)
+    if pa.refcount[] > 0
```

Comment:
How would it be possible for the refcount to be 0? That would mean even `pa` doesn't hold a reference?

Reply:
```
julia> Threads.Atomic()
Base.Threads.Atomic{Int64}(0)
```
So `pa.refcount[]` is `0` by default (I will add a comment on this).

Reply:
Ah OK. :-D

Maybe it would be more intuitive to start at `1`?

Reply:
OK - I will change it.